swagger.yaml

```diff
@@ -17,23 +17,39 @@ info:
     name: Open Service Broker API
     url: 'https://www.openservicebrokerapi.org/'
     email: open-service-broker-api@googlegroups.com
-  version: v2.13
+  version: master - might contain changes that are not yet released
+
+host: localhost:3000
```

Comment:
Should we have a `host` in here?

Reply:
I saw `servers` is contained in `openapi.yaml`, so I added this `host` field to keep up with it.